ATP.BusinessLogicLayer/ATP.BusinessLogicLayer.csproj

```diff
@@ -6,4 +6,8 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\ATP.DataAccessLayer\ATP.DataAccessLayer.csproj" />
+  </ItemGroup>
```

Comment:
Why it sees the data access layer, the domain or the business logic layer should not know anything about the data access layer.

Reply:
Again